src/docs/product/performance/filters-display/index.mdx

```diff
@@ -44,11 +44,11 @@ When you select "All Transactions", the table includes all transactions across a
 
 <Note>
 
-This feature is in beta and is only available if your organization is participating in the limited release of [metrics](https://docs.sentry.io/product/sentry-basics/metrics/). Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup). If you are not participating in the limited release, search condition filters are used instead of transaction search.
+This feature is in beta and is only available if your organization is participating in the limited release of [dynamic sampling](https://docs.sentry.io/product/sentry-basics/sampling/). Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup). If you are not participating in the limited release, search condition filters are used instead of transaction search.
 
 </Note>
 
-When viewing the [metrics](https://docs.sentry.io/product/sentry-basics/metrics/) (processed) dataset, the search bar is used to filter by transaction name only, rather than to search for transactions matching the condition filters.
+When viewing the [processed transactions](https://docs.sentry.io/product/sentry-basics/sampling/) dataset, the search bar is used to filter by transaction name only, rather than to search for transactions matching the condition filters.
```

Comment:
```suggestion
When viewing the [processed transactions](/product/sentry-basics/sampling/) dataset, the search bar is used to filter by transaction name only, rather than to search for transactions matching the condition filters.
```